caas/kubernetes/provider/specs/v2_test.go

```diff
@@ -539,6 +587,9 @@ password: shhhh`[1:],
 							Plural:   "tfjobs",
 							Singular: "tfjob",
 						},
+						Conversion: &apiextensionsv1beta1.CustomResourceConversion{
```

Comment:
Is this the only new field? I thought there was another we could add to the test?